Cargo.toml

```diff
@@ -154,22 +154,22 @@ ethers-middleware = { version = "2.0", default-features = false }
 ethers-solc = { version = "2.0", default-features = false }
 
 ## alloy
-alloy-consensus = { git = "https://github.com/alloy-rs/alloy" }
-alloy-eips = { git = "https://github.com/alloy-rs/alloy" }
-alloy-genesis = { git = "https://github.com/alloy-rs/alloy" }
-alloy-json-rpc = { git = "https://github.com/alloy-rs/alloy" }
-alloy-network = { git = "https://github.com/alloy-rs/alloy" }
-alloy-node-bindings = { git = "https://github.com/alloy-rs/alloy" }
-alloy-providers = { git = "https://github.com/alloy-rs/alloy" }
-alloy-pubsub = { git = "https://github.com/alloy-rs/alloy" }
-alloy-rpc-client = { git = "https://github.com/alloy-rs/alloy" }
-alloy-rpc-trace-types = { git = "https://github.com/alloy-rs/alloy" }
-alloy-rpc-types = { git = "https://github.com/alloy-rs/alloy" }
-alloy-signer = { git = "https://github.com/alloy-rs/alloy" }
-alloy-transport = { git = "https://github.com/alloy-rs/alloy" }
-alloy-transport-http = { git = "https://github.com/alloy-rs/alloy" }
-alloy-transport-ipc = { git = "https://github.com/alloy-rs/alloy" }
-alloy-transport-ws = { git = "https://github.com/alloy-rs/alloy" }
+alloy-consensus = { git = "https://github.com/alloy-rs/alloy", rev = "785c667" }
+alloy-eips = { git = "https://github.com/alloy-rs/alloy", rev = "785c667" }
+alloy-genesis = { git = "https://github.com/alloy-rs/alloy", rev = "785c667" }
+alloy-json-rpc = { git = "https://github.com/alloy-rs/alloy", rev = "785c667" }
+alloy-network = { git = "https://github.com/alloy-rs/alloy", rev = "785c667" }
+alloy-node-bindings = { git = "https://github.com/alloy-rs/alloy", rev = "785c667" }
+alloy-providers = { git = "https://github.com/alloy-rs/alloy", rev = "785c667" }
+alloy-pubsub = { git = "https://github.com/alloy-rs/alloy", rev = "785c667" }
+alloy-rpc-client = { git = "https://github.com/alloy-rs/alloy", rev = "785c667" }
+alloy-rpc-trace-types = { git = "https://github.com/alloy-rs/alloy", rev = "785c667" }
+alloy-rpc-types = { git = "https://github.com/alloy-rs/alloy", rev = "785c667" }
+alloy-signer = { git = "https://github.com/alloy-rs/alloy", rev = "785c667" }
+alloy-transport = { git = "https://github.com/alloy-rs/alloy", rev = "785c667" }
+alloy-transport-http = { git = "https://github.com/alloy-rs/alloy", rev = "785c667" }
+alloy-transport-ipc = { git = "https://github.com/alloy-rs/alloy", rev = "785c667" }
+alloy-transport-ws = { git = "https://github.com/alloy-rs/alloy", rev = "785c667" }
```

Comment:
this is necessary for evm-inspectors right?

Reply:
yeah, should we start publishing the rpc types?
this would make it a lot easier

Reply:
We will with the rest of alloy, this is ok for now